src/android/toga_android/layout.py

```diff
@@ -1,19 +1,19 @@
 
-class TogaLayout(extends=android.view.ViewGroup):
-    @super({context: android.content.Context})
+class TogaLayout:
+    # TODO: Extend `android.view.ViewGroup`. Provide app as `context`.
     def __init__(self, context, interface):
         self.interface = interface
 
     def shouldDelayChildPressedState(self) -> bool:
         return False
 
-    def onMeasure(self, width: int, height: int) -> void:
+    def onMeasure(self, width: int, height: int):
```

Comment:
why get rid of the return type? this should be None, right?

Reply:
For what it's worth, the `-> void` is a relic of the Android VOC approach, where these annotations are used to generate Java bytecode. I don't mind making this `-> None`, and will do so.